optimum/intel/openvino/modeling_diffusion.py

```diff
@@ -636,24 +642,31 @@ def __init__(
         parent_model: OVBaseModel,
         ov_config: Optional[Dict[str, str]] = None,
         model_name: str = "text_encoder",
+        model_dir: str = None,
```

Comment:
is it set anywhere and if no what about removing it (potentially from `OVModelPart`)